conf/logback.xml

```diff
@@ -15,7 +15,7 @@
   <logger name="play" level="INFO" />
   <logger name="application" level="DEBUG" />
 
-  <root level="ERROR">
+  <root level="INFO">
     <appender-ref ref="STDOUT" />
```

Comment:
(I've noticed us logging to stdout in many places and think we need to deprecate the practice.)